# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../perl/pthreads_hotfix.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From efcebb94c92d3810f5500b035bb7454612a4204f Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 23 Apr 2014 09:02:25 +0200
Subject: [PATCH] It's -lpthread not -lpthreads

Otherwise linking fails with missing pthread symbols when building with
with -Dusethreads
---
 cnf/configure_libs.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cnf/configure_libs.sh b/cnf/configure_libs.sh
index e970947..651c9d8 100644
--- a/cnf/configure_libs.sh
+++ b/cnf/configure_libs.sh
@@ -53,7 +53,7 @@ if not hinted 'perllibs'; then
 				appendvar '_libs' "$i" ;;
 			-ldl)
 				test "$usedl" != 'undef' && appendvar '_libs' "$i" ;;
-			-lpthreads)
+			-lpthread)
 				test "$usethreads" != 'undef' && appendvar '_libs' "$i" ;;
 			# For a static build, -lgdbm and friends are assumed to be in ext.libs
 		esac
-- 
1.9.3

